### DISCUSSION TITLE
lateral dose distribution running for TsImageCube

### DISCUSSION THREAD

**Discussion by DararaiPakraypet** [OpenTOPAS/OpenTOPAS#91] [Feature-Request] [answered]
### Feature request

Dear Developer, 

I would like to run a simulation to get the lateral dose distribution in a 1D format along z axis using TsImageCube. How can I achieve this? Do you have any recommendations? I have the phantom parameters as follows:
i:Ge/Patient/NumberOfVoxelsX = 271
i:Ge/Patient/NumberOfVoxelsY = 190
iv:Ge/Patient/NumberOfVoxelsZ = 1 775
d:Ge/Patient/VoxelSizeX       = 1.579 mm
d:Ge/Patient/VoxelSizeY       = 1.579 mm
dv:Ge/Patient/VoxelSizeZ     = 1 2.207 mm

Best regard,
DP

### TOPAS category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by icmeyer** [comment, marked as answer]
What format is your image in? Is this a DICOM file or `.dat` or something other format? 

One correction on what you have so far: Both `NumberOfVoxelsZ` and `VoxelSizeZ` should be changed from vector arguments (`dv`) to only singleton (`i` or `d`)

You can mostly follow the [DoseToCT](https://opentopas.readthedocs.io/en/latest/examples-docs/Patient/DoseToCT.html) example from the documentation. For setting the materials appropriately please refer to the [Patient Components](https://opentopas.readthedocs.io/en/latest/parameters/geometry/patient.html) documentation. The dose will be output in either ASCII or binary and can be converted and plotted using a tool like [topas2numpy](https://github.com/davidchall/topas2numpy). It looks like you are just at the start so please post any more questions that come up as you put your simulation together.

**Reply by DararaiPakraypet** [reply]
Thank you so much! I can score the lateral dose in my simulation work.
